modules/bison/3.8.2/overlay/BUILD.bazel

```diff
@@ -0,0 +1,495 @@
+    "lib/unitypes.h": "lib/unitypes.in.h",
+    "lib/uniwidth.h": "lib/uniwidth.in.h",
+    "lib/wchar.h": "lib/wchar.in.h",
+    "lib/wctype.h": "lib/wctype.in.h",
+}
+
+USES_DIRECT_REPLACEMENTS = [
+    "lib/malloc/scratch_buffer.gl.h",
+]
+
+[
+    genrule(
+        name = hdr[:-len(".h")] + "_h",
+        srcs = [template] + select({
+            "@platforms//os:macos": MACOS_INLINE_VARS.values(),
+            "//conditions:default": LINUX_INLINE_VARS.values(),
+        }),
+        outs = [hdr],
+        cmd = select({
+            "@platforms//os:macos": sed_command(
+                automake_vars = MACOS_AUTOMAKE_VARS,
+                delete_vars = MACOS_DELETE_VARS,
+                direct_vars = MACOS_DIRECT_VARS,
+                inline_vars = MACOS_INLINE_VARS,
+                sed = "@sed",
+                template = template,
+                use_direct_vars = True if hdr in USES_DIRECT_REPLACEMENTS else False,
+            ),
+            "//conditions:default": sed_command(
+                automake_vars = LINUX_AUTOMAKE_VARS,
+                delete_vars = LINUX_DELETE_VARS,
+                direct_vars = LINUX_DIRECT_VARS,
+                inline_vars = LINUX_INLINE_VARS,
+                sed = "@sed",
+                template = template,
+                use_direct_vars = True if hdr in USES_DIRECT_REPLACEMENTS else False,
+            ),
+        }),
+        cmd_bat = select({
+            "@platforms//os:macos": sed_command(
+                automake_vars = MACOS_AUTOMAKE_VARS,
+                delete_vars = MACOS_DELETE_VARS,
+                direct_vars = MACOS_DIRECT_VARS,
+                inline_vars = MACOS_INLINE_VARS,
+                is_windows = True,
+                sed = "@sed",
+                template = template,
+                use_direct_vars = True if hdr in USES_DIRECT_REPLACEMENTS else False,
+            ),
+            "//conditions:default": sed_command(
+                automake_vars = LINUX_AUTOMAKE_VARS,
+                delete_vars = LINUX_DELETE_VARS,
+                direct_vars = LINUX_DIRECT_VARS,
+                inline_vars = LINUX_INLINE_VARS,
+                is_windows = True,
+                sed = "@sed",
+                template = template,
+                use_direct_vars = True if hdr in USES_DIRECT_REPLACEMENTS else False,
+            ),
+        }),
+        tools = ["@sed"],
+    )
+    for hdr, template in GENERATED_SRCS.items()
+]
+
+copy_file(
+    name = "config_h_src",
+    src = select({
+        "@platforms//os:macos": "config_macos.h",
+        "//conditions:default": "config_linux.h",
+    }),
+    out = "lib/config.h",
+    tags = ["manual"],
+)
+
+write_file(
+    name = "configmake_h_src",
+    out = "lib/configmake.h",
+    content = [
+        "#define LOCALEDIR \"\"",
+        "#define PKGDATADIR \"bison/data\"",
+        "",
+    ],
+    newline = "unix",
+    tags = ["manual"],
+)
+
+config_setting(
+    name = "cc_compiler_clang",
```

Comment:
Can you use the settings in @rules_cc//cc/compiler instead?

Reply:
Done!